### PR TITLE
mobile version

### PR DESCRIPTION
**Pull request by Obote**
### What does this PR do?

- Make sure that there are no linter errors
- Use correct GitHub Flow
- Understand how to parse a Figma design to create a UI.
- Use Flexbox to place elements in the page.
- Use CSS Grid to place elements in the page.
- Build a personal portfolio site.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,6 +23,97 @@ <h2>Software Developer</h2>
...
+        <h3>Multi-Post Stories</h3>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="stack-container">
+            <li class="border-right padding-right">css</li>
+            <li class="border-right padding-right">html</li>          
+            <li class="border-right padding-right">bootstrap</li>         
+            <li class="padding-right">Ruby</li>
+        </ul>
+        <button type="button" class="project-button">See Project</button>
+    </section>
```
Comment (ZeenatLawal) [inline index.html:116]: - [ ] The whole works section should be in a container (`section`) which should have a grid display.
- [ ] I think each project card should be aligned with flex as stated in the requirements.
- [ ] The See Project buttons in this section should follow the button styling: Active, Hover and Pressed (see design)
- [ ] Kindly make sure this section has the proper background image and font as defined in the 
template

File: index.html [hunk 2]
```diff
@@ -23,6 +23,97 @@ <h2>Software Developer</h2>
...
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="stack-container">
+            <li class="border-right padding-right">css</li>
+            <li class="border-right padding-right">html</li>          
+            <li class="border-right padding-right">bootstrap</li>         
+            <li class="padding-right">Ruby</li>
+        </ul>
+        <button type="button" class="project-button">See Project</button>
+    </section> 
     </main>
```
Comment (ZeenatLawal) [inline index.html:117]: - [ ] The entire About myself section is missing and this is a major requirement, kindly work on it.

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @Obote , 

Great job so far!:clap: :clap:
## To Highlight 🎊 🎊 
- GitHub flow is followed. ✔️
- PR has a title and summary. ✔️

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Obote) [reply index.html:117]: The about myself section is on top, I think your doing the review based on a different template not template 4

**Review by Leon-Mbegera — APPROVED** [review]
Hi @Obote , 

# Approved ✔

- Well done for implementing all the requested changes 👍 

Your project is complete!  There is nothing else to say other than... it's time to merge it  and move forward to the next stage. :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**